platform/darwin/src/MGLStyle.h

```diff
@@ -177,6 +177,14 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 /**
  Returns a layer that conforms to `MGLStyleLayer` if any layer with the given
  identifier was found.
+ 
+ @note Layer identifiers are not guaranteed to exist across styles or different
+    versions of the same style. Applications that use this API must set the style 
+    URL to an explicitly versioned style using a convenience method like 
+    `+[MGLStyle outdoorsStyleURLWithVersion:]`, `MGLMapView`'s “Style URL” 
+    inspectable in Interface Builder, or a manually constructed NSURL. This 
```

Comment:
Nit: put `NSURL` in backticks (for consistency with SDK classes that would get autolinked, even if this one doesn’t get linked).